### PR TITLE
4 More Playbooks - 1 updated Playbook

### PR DESCRIPTION
**Pull request by koipond**
I added the following playbooks

-Hardholder
-hocus
-meastro D
-savvyhead

updated Angel to include kit roll.

### CODE REVIEW

File: playbooks/angel.json [hunk 1]
```diff
@@ -6,4 +6,11 @@
         "Hit": "You put some guts back, heal 1 segment but you gotta openbrain now and hope the Pyschic Maelstrom plays nice.",
```
Comment (gamefiend) [inline playbooks/angel.json:6]: typo: Pyshic -> Psychic

File: playbooks/angel.json [hunk 2]
```diff
@@ -6,4 +6,11 @@
         "Hit": "You put some guts back, heal 1 segment but you gotta openbrain now and hope the Pyschic Maelstrom plays nice.",
         "Crit": "You put lots of guts back. heal 1 segment."
        },
-]
\ No newline at end of file
+       { 
+        "Name": "!kit",
+        "Full": "Angel Kit - Stabalize at 9:00 or past",
```
Comment (gamefiend) [inline playbooks/angel.json:11]: Stabalize -> Stabilize

File: playbooks/maestrod.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+[ 
+    { 
+        "Name": "!everybodyeats",
+        "Full": "Everbody eats, even that guy",
+        "Miss": "Ask the MC 1 question about someone, but that person knows you're looking in on them",
+        "Hit": "Ask the MC 1 question.\n* How are they doing? what's up with them?\n* What or who do they love best\n* Who do they know, like and/or trust?\n* When next should I expect to see them?\n* How could I get to them, physically or emotionally?",
```
Comment (gamefiend) [inline playbooks/maestrod.json:6]: "What or who do they love best" -> "What or who do they love best?"

File: playbooks/maestrod.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+[ 
+    { 
+        "Name": "!everybodyeats",
+        "Full": "Everbody eats, even that guy",
+        "Miss": "Ask the MC 1 question about someone, but that person knows you're looking in on them",
+        "Hit": "Ask the MC 1 question.\n* How are they doing? what's up with them?\n* What or who do they love best\n* Who do they know, like and/or trust?\n* When next should I expect to see them?\n* How could I get to them, physically or emotionally?",
+        "Crit": "Ask the MC 3 questions.\n* How are they doing? what's up with them?\n* What or who do they love best\n* Who do they know, like and/or trust?\n* When next should I expect to see them?\n* How could I get to them, physically or emotionally?"
```
Comment (gamefiend) [inline playbooks/maestrod.json:7]: "What or who do they love best" -> "What or who do they love best?"

File: playbooks/savvyhead.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
+[ 
+    { 
+        "Name": "!thingsspeak",
+        "Full": "Things Speak",
+        "Miss": "This is what happens when you play with the Psychich Maelstrom. It plays with your head. Open your brain, but you've already missed the roll",
```
Comment (gamefiend) [inline playbooks/savvyhead.json:5]: Psychich -> Psychic

File: playbooks/savvyhead.json [hunk 6]
```diff
@@ -0,0 +1,17 @@
+[ 
+    { 
+        "Name": "!thingsspeak",
+        "Full": "Things Speak",
+        "Miss": "This is what happens when you play with the Psychich Maelstrom. It plays with your head. Open your brain, but you've already missed the roll",
+        "Hit": "Ask the MC 1 question.\n* Who handled this last befor me?\n* Who made this?\n* What strong emotions have been most recently nearby this?\n* What words have been said most recently nearby this?\n* What has been done most recently with this, or to this?\n* What's wrong with this, and how might I fix it?",
```
Comment (gamefiend) [inline playbooks/savvyhead.json:6]: "Ask the MC 1 question." -> "Ask the MC 1 question:"

File: playbooks/savvyhead.json [hunk 7]
```diff
@@ -0,0 +1,17 @@
+[ 
+    { 
+        "Name": "!thingsspeak",
+        "Full": "Things Speak",
+        "Miss": "This is what happens when you play with the Psychich Maelstrom. It plays with your head. Open your brain, but you've already missed the roll",
+        "Hit": "Ask the MC 1 question.\n* Who handled this last befor me?\n* Who made this?\n* What strong emotions have been most recently nearby this?\n* What words have been said most recently nearby this?\n* What has been done most recently with this, or to this?\n* What's wrong with this, and how might I fix it?",
+        "Crit": "Ask the MC 3 questions.\n* Who handled this last befor me?\n* Who made this?\n* What strong emotions have been most recently nearby this?\n* What words have been said most recently nearby this?\n* What has been done most recently with this, or to this?\n* What's wrong with this, and how might I fix it?"
```
Comment (gamefiend) [inline playbooks/savvyhead.json:7]: "Ask the MC 1 question." -> "Ask the MC 1 question:"

**Review by gamefiend — APPROVED** [review]
Looks good, just some typos to fix and will start working on incorporating playbooks ASAP!

**Comment by olleolleolle** [comment]
@gamefiend Maybe you can merge, and then we can collectively fix any typos?

**Pull request merged**